extension/popup/popup.js

```diff
@@ -1,17 +1,30 @@
 import { createBookmark } from './services/BookmarkService.js';
-import { getActiveGroupInfo, getTabsForCurrentWindow } from './services/TabService.js';
+import { getActiveGroupInfo, getTabsForCurrentWindow, generateBookmarkName } from './services/TabService.js';
+
+const getBookmarkTitle = async (payload) => {
+  return (document.getElementById("inputTitle").value)? document.getElementById("inputTitle").value: await generateBookmarkName(payload.tabs.length);   
+}
+
+const awake = async () => {
+  const groupInfo = await getActiveGroupInfo();
+  if (groupInfo && groupInfo.title) {
+    document.getElementById("inputTitle").value = groupInfo.title
+  } else {
+    document.getElementById('group-btn').disabled = true;
```

Comment:
Shouldn't we explicitly enable this button if it is a group in the if clause above?